src/test/scala/mesosphere/marathon/integration/setup/MarathonTest.scala

```diff
@@ -878,8 +878,10 @@ trait MarathonClusterTest extends Suite with StrictLogging with ZookeeperServerT
     super.afterAll()
   }
 
-  def nonLeader(leader: ITLeaderResult): MarathonFacade = {
-    marathonFacades.find(!_.url.contains(leader.port.toString)).get
+  def nonLeader(): MarathonFacade = {
```

Comment:
idea is not finding any usages of this method... and since you did not have to alter any usages here and the code still compiles, do we need this method ... ?

Reply:
I was wondering the same. I did use it for an integration test that did not work. Since it is not used at all I'll just remove it.